docs/app/Examples/modules/Sticky/Types/StickyAdjacentContextExample.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import { Grid, Header, Image, Rail, Segment, Sticky } from 'semantic-ui-react'
+import _ from 'lodash'
+
+const Placeholder = () => (
+  <Image
+    className='wireframe paragraph image'
```

Comment:
BTW, these classes are from SUI docs, so I don't sure that they defined in our CSS. @mariolamacchia can you check? Seems they are cruft.